src/pip/__init__.py

```diff
@@ -4,7 +4,7 @@
     from typing import List, Optional
 
 
-__version__ = "20.2"
+__version__ = "20.3.dev0"
```

Comment:
Will need to fix this

Reply:
Not really -- this is based off the 20.2 tag, and the 20.3.dev0 matches what master says currently.